.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Context
+
+(Is your feature request related to a problem?)  
+*Ex: I get frustrated when having to use nested `if` conditions where a `elif` could be used*
+
+# Description of the feature
+
+(What do you want to see happening)  
+*Ex: I would like the keyword `elif` to be added to Inmanta Language*
+
+# Describe alternatives you've considered
```

Comment:
I don't know if this is appropriate. It invites a deeper discussion, but it also greatly increases the effort required to make a request?

Reply:
Should I add "if any"? To make it feel less like a requirement

Reply:
(Did it in https://github.com/inmanta/inmanta/pull/2416/commits/2e2f1f6a155d159d30daa2953c86835f4bde579f)